apps/web/public/static/locales/en/common.json

```diff
@@ -774,5 +774,16 @@
   "impersonate_user_tip":"All uses of this feature is audited.",
   "impersonating_user_warning":"Impersonating username \"{{user}}\".",
   "impersonating_stop_instructions": "<0>Click Here to stop</0>.",
+  "setting_up_zapier": "Setting up your Zapier integration",
+  "generate_api_key": "Generate Api Key",
+  "your_unique_api_key": "Your unique API key",
+  "copy_safe_api_key": "Copy this API key and save it somewhere safe. If you lose this key you have to generate a new one.",
+  "log_into_zapier_account": "1. Log into your Zapier account and create a new Zap.",
+  "select_cal_trigger_app": "2. Select Cal.com as your Trigger app. Also choose a Trigger event.",
+  "choose_account_enter_key": "3. Choose your account and then enter your Unique API Key.",
+  "test_your_trigger": "4. Test your Trigger.",
+  "you_are_set": "5. You're set!",
+  "install_zapier_app": "Please first install the Zapier App in the app store.",
+  "go_to_app_store": "Go to App Store",
```

Comment:
Should we name space these with `"zapier_`? At least for the steps.

Reply:
Should the steps be a separate translation for each line? I agree with the above either way